src/ucp/core/ucp_ep.h

```diff
@@ -357,9 +357,10 @@ enum {
                                              extra MSGs. */
     UCP_WIREUP_SOCKADDR_CD_CM_ADDR        /* Client data contains address for CM
                                              based wireup, there is only iface
-                                             and ep address, remote device
-                                             address is provided by CM and full
-                                             address have to be merged locally. */
+                                             and ep address of transport lanes,
```

Comment:
of all lanes?

Reply:
until we don't have AUX path for CMs - yes